### DISCUSSION TITLE
멀티 플레이를 실현할 수 있었지만, 아바타가 움직이면 아바타는 사라집니다

### DISCUSSION THREAD

**Discussion by noil-zepe** [naverz/zepeto-studio-kor#1200] [5. Q&A]
멀티 플레이 튜토리얼 동영상을 보면서 멀티 플레이를 구현했습니다.

QR 코드를 사용하여 두 개의 계정으로 테스트를 플레이했습니다.

문제없이 같은 방에 들어갈 수 있었지만 아바타를 이동하거나 점프하면 아바타가 사라집니다.
unity의 테스트 플레이에서도 같은 현상이 발생합니다.

무엇이 문제입니까.. 가르쳐 주시면 감사하겠습니다.
잘 부탁드립니다.

문제없이 실행되는 모습
![スクリーンショット 2022-08-22 9 54 13](https://user-images.githubusercontent.com/102274326/185819481-364cbe75-0c7c-48a1-87fc-2f0266c18a5c.png)

unity 테스트 플레이에서 아바타가 사라진 후의 모습
![スクリーンショット 2022-08-22 9 58 02](https://user-images.githubusercontent.com/102274326/185819699-5ec8e8f2-2411-4edc-85a1-add3fa0787ab.png)

해당 스크립트
![スクリーンショット 2022-08-22 10 00 14](https://user-images.githubusercontent.com/102274326/185819756-f72cf28f-1bcd-4cc2-a306-4242920b7b9d.png)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 첨부해주신 부분은 클라이언트 코드인데 서버쪽에서도 "onChangedState" 메세지를 받았을때 상태를 갱신하는 코드가 잘 구현되었는지를 보시면 좋을 것 같습니다.

멀티플레이 샘플 중에서
서버 코드 :  https://github.com/naverz/zepeto-multiplay-example/blob/main/Assets/World.multiplay/index.ts 이 부분과
클라이언트 코드 : https://github.com/naverz/zepeto-multiplay-example/blob/main/Assets/ZepetoScripts/ClientStarter.ts
이 부분을 참고하여 주세요.
감사합니다.

**Reply by noil-zepe** [reply]
동영상을 보면서 수타했기 때문에, 실수하고 있었던 것 같다 .. !
 감사합니다 ^^
